README.md

```diff
@@ -10,6 +10,42 @@ If there is anything you are looking for that is not covered here you should ref
 
 ----
 
+### File structure ###
+
+You should not have multiple classes in the same file, each class should have its own file. 
+
+The following list should be the standard struct/class organization of all your Swift files, in this specific order:
+
+* Static variables
+* View Lifecycle (if available)
```

Comment:
When we say view lifecycle it's to combine all the view functions at the top since they most of the time are the most important things to look at in your view controller. They only apply to view controllers that's why we specify if available, maybe we should say `view controllers only`.

The organization you suggest looks good to me though, but you forgot protocol implementation that should be an extension after the class declaration.